src/extension.ts

```diff
@@ -34,21 +32,21 @@ interface IResult {
 /**
  * Show message in output channel.
  */
-function showOutput(msg: string): void {
-  msg = msg.toString()
+function showOutput(message: string): void {
+  message = message.toString()
 
   if (!output) {
     output = vscode.window.createOutputChannel('Remark')
   }
 
   output.clear()
   output.appendLine('[Remark]')
-  output.append(msg)
+  output.append(message)
   output.show()
 }
 
-function getPlugins(list: string[]): Promise<IPlugin[]> {
-  const root = vscode.workspace.rootPath || ''
+async function getPlugins(list: string[]): Promise<IPlugin[]> {
+  const root = vscode.workspace.rootPath ?? ''
```

Comment:
VS Code is Electron right, or not? Will it support such new features?
Should this XO rule be turned off?

Reply:
This is transpiled by TypeScript.